web/scripts/users/profileComponent.js

```diff
@@ -11,6 +11,8 @@ module.exports = {
       template: `
       <div id="profile-root">
         <div id="user-profile">
+          <h2 style="text-align: center">Profile Information</h2>
```

Comment:
Can you please move the styling into the respective .less files instead of having the styling in the component.

Reply:
👍